tests/ExportTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
please just implement new test case inside the FixtureTraitTest, which will call `unset($this->globalExportMode);` inside